packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -162,67 +176,48 @@ class DDSTabsExtended extends StableSelectorMixin(LitElement) {
     return tabItems;
   }
 
-  updated() {
+  updated(_changedProperties) {
+    const { _isMobileVersion } = this;
     this._isLTR = window.getComputedStyle(this).direction === 'ltr';
     this._activeTabIndex = parseInt(this._activeTab, 10);
 
-    this._tabItems.map((tab, index) => {
+    this._tabItems.forEach((tab, index) => {
       (tab as DDSTab).selected = index === this._activeTabIndex;
       (tab as DDSTab).setIndex(index);
```

Comment:
Is this something we really want to do _every time_ we go through the render lifecycle?

Reply:
Related: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/10650/files#r1265792886